libs/nestjs-admin/src/defaultAdmin.module.ts

```diff
@@ -0,0 +1,15 @@
+import { Module } from '@nestjs/common'
+import { CoreAdminModule } from './coreAdmin.module'
+import { AuthAdminModule } from './authAdmin.module'
+import DefaultAdminSite from './adminSite'
+import AdminUserEntity from './adminUser.entity'
+
+@Module({
+  imports: [CoreAdminModule, AuthAdminModule],
+  exports: [CoreAdminModule, AuthAdminModule],
```

Comment:
that's where you should instantiate the AdminCoreModuleFactory and AdminAuthModuleFactory, so that there's no default exposed